### PR TITLE
Updated Azure to latest stable build (v1.0.3)

### PR DESCRIPTION
**Pull request by JakeNarkizian**
Minor changes to the codebase

### CODE REVIEW

**Comment by JakeNarkizian** [comment]
@hannes-ucsc ready

File: src/toil/jobStores/azureJobStore.py [hunk 1]
```diff
@@ -713,12 +715,12 @@ def toItem(self, chunkSize=maxAzureTablePropertySize):
 def retryOnAzureTimeout(exception):
     timeoutMsg = "could not be completed within the specified time"
     busyMsg = "Service Unavailable"
-    return isinstance(exception, WindowsAzureError) and (timeoutMsg in str(exception)
-        or busyMsg in str(exception))
+    return isinstance(exception, AzureException) and (timeoutMsg in str(exception)
+                                                      or busyMsg in str(exception))
```
Comment (hannes-ucsc) [inline src/toil/jobStores/azureJobStore.py]: wrap **before** `and` and keep the disjunction on a line. When wrapping expressions (which are trees), prefer keeping the leafs together on a line and instead wrap at higher levels in the tree.

**Comment by hannes-ucsc** [comment]
Is there a ticket for this? If so, it should be referenced here. If not, the branch name should have been `features/…` instead of `issues/…` but it's ok to leave it as is for this PR. I never mentioned this explicitly as I made that convention up a few weeks ago and never told anyone.

Branch needs to be rebased and see my line notes.

**Comment by JakeNarkizian** [comment]
@hannes-ucsc

**Pull request merged**